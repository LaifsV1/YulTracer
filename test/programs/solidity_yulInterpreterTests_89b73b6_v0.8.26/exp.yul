{
  mstore(0, exp(3,not(1)))

  // Added memory dump:
  printhex(mload(0))
}
// ----
// Trace:
// Memory dump:
//      0: 8e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e39
// Storage dump:
// Transient storage dump:
