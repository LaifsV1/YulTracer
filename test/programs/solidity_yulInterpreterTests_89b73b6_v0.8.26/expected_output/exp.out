8e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e38e39
